### PR TITLE
change Graphene recipes

### PR DESCRIPTION
**Pull request by Dream-Master**
add new tiered recipes

### CODE REVIEW

**Comment by GlodBlock** [comment]
tho this recipe is too expensive compared with the blast one which takes less time and is more effective in turing graphite into graphene. at least the graphene output should be doubled or tripled.

**Comment by Dream-Master** [comment]
The blast recipe are removed. Look at the original recipe . It was hv 480 ticks at 240 gives a small dust.

**Comment by Dream-Master** [comment]
The blast recipe was a joke and overseen for years

**Comment by Dream-Master** [comment]
I can add old recipe back (with the small dust) but ebf recipe will be disabled

**Comment by GlodBlock** [comment]
the old assembler recipe is meaningless. but could you please doubled the output so we still can turn graphite into graphene by 1:1 in zpm tier with mk2 fusion.

File: src/main/java/gregtech/loaders/postload/GT_MachineRecipeLoader.java [hunk 1]
```diff
@@ -447,7 +447,11 @@ public void run() {
...
         GT_Values.RA.addAssemblerRecipe(new ItemStack[]{GT_ModHandler.getIC2Item("carbonMesh", 4L), GT_OreDictUnificator.get(OrePrefixes.foil, Materials.Zinc, 16L),  GT_Utility.getIntegratedCircuit(1)}, GT_Values.NF, ItemList.Component_Filter.get(1L), 1600, 30);
         GT_Values.RA.addAssemblerRecipe(new ItemStack[]{GT_OreDictUnificator.get(OrePrefixes.wireFine, Materials.Steel, 64), GT_OreDictUnificator.get(OrePrefixes.foil, Materials.Zinc, 16L),  GT_Utility.getIntegratedCircuit(1)}, Materials.Plastic.getFluid(144), ItemList.Component_Filter.get(1), 1600, 16);
-        GT_Values.RA.addAssemblerRecipe(GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Graphite, 8), GT_OreDictUnificator.get(OrePrefixes.foil, Materials.Silicon, 1), Materials.Glue.getFluid(250L), GT_OreDictUnificator.get(OrePrefixes.dustSmall, Materials.Graphene, 1), 480, 240);
+        GT_Values.RA.addAssemblerRecipe(new ItemStack[]{GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Graphite, 32), ItemList.Circuit_Silicon_Wafer.get(1L), GT_Utility.getIntegratedCircuit(1)}, Materials.Glue.getFluid(1000L), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Graphene, 1), 1200, 480);
```
Comment (boubou19) [inline src/main/java/gregtech/loaders/postload/GT_MachineRecipeLoader.java:450]: From memory, you need 7 motors for the assembler: 3 hidden in the robot arm and 4 in the conveyors. A motor is 4 x4 graphene wires, so 8 ingots / motor. That's 48 graphene dusts. The HV recipe will take 600 ticks in EV, so 30s each. That's 24 min for the dusts.

Meanwhile, the EV recipe will do 2 dusts in 900 ticks, so 450 ticks per dust (22.5s) for a total of 18min. Do we really need that HV recipe given graphene is an IV material?

**Comment by iouter** [comment]
so, this means that an iv motor needs 256 graphite dusts at iv early. crazy.

**Comment by boubou19** [comment]
isn't 1:1 for the ratio graphene wire/dust? and a motor is 16 graphene wires no? how do you reach 4 stacks?

**Comment by iouter** [comment]
> isn't 1:1 for the ratio graphene wire/dust? and a motor is 16 graphene wires no? how do you reach 4 stacks?

each graphene needs 16 graphite if use the glowstone wafer.

**Comment by boubou19** [comment]
@Dream-Master a ratio 16:1 for a wire is too big with those recipe times. That's 4 stacks of dusts. It's insanely long: imagine taking 14h56min just to bake the the amount of graphene dusts needed for the IV assembler with the HV recipe.  I suggest to be a 2:1 ratio for the conversion dust -> wire, that would take 56 mins with the HV recipe, much more reasonable.

**Comment by boubou19** [comment]
I just talked to Dream, and it seems the ratio between graphene dust and graphene wire will stay at 1:1.

**Comment by Dream-Master** [comment]
> I just talked to Dream, and it seems the ratio between graphene dust and graphene wire will stay at 1:1.

The wire recipe stay untouched

**Review by boubou19 — APPROVED** [review]

**Comment by Glease** [comment]
We need to revert or change this as well. A single diamond vein grant a little over 20 stacks of graphite. Your first IV assembler (required to use second "improved" recipe) require 16 IV motor == 256 graphene wire == 256 graphene dust == 64 stacks of graphite. Mine two or three diamond vein to be able to just make your first IV machine, and another 1 vein for every IV machine made? Not wise.

**Comment by Dream-Master** [comment]
No more ebf recipe this was an exploit and I won’t add it back

**Comment by Dream-Master** [comment]
We can thinking of the amount of Graphit will used

**Comment by Glease** [comment]
I will not accept 64 stacks of graphite whatsoever. This just make no sense at all and make IV stage impossibly hard to enter.

**Comment by Sphyix** [comment]
> No more ebf recipe this was an exploit and I won’t add it back

ebf recipe was too cheap, but made more sense than what we have now
changing it by making it more expensive in terms of power and a bit more graphite would have been ok

**Comment by Dream-Master** [comment]
Nope ebf recipe make no sense at all compared to rl graphene recipes

**Comment by Sphyix** [comment]
Directly from google:
"Growing graphene
For the most part, scientists synthesize graphene through a process called chemical vapor deposition, in which they first heat a sample of copper foil and then deposit onto it a combination of carbon and other gases."

Where is graphite used? And where are wafers used then?
Let's use copper foil, carbon and some gas then

**Pull request merged**